src/com.company/backtracking/UniquePaths.java

```diff
@@ -0,0 +1,74 @@
+package backtracking;
+
+import java.util.*;
+
+public class UniquePaths {
+
+    int[] rowIndex =  new int[] {-1, 0, 0, 1};
+    int[] colIndex = new int[]{0, -1, 1, 0};
+    static int count = 0;
+    private int solve(int[][] maze) {
+        int[] start = new int[] {-1, -1};
+        for(int i = 0; i< maze.length; i++){
```

Comment:
Provide proper alignment. Use command + shift + enter.

Reply:
formatted the code